### ISSUE TITLE
Add CircleCI to this repo

### ISSUE THREAD

**Issue by codito** [spekt/junit.testlogger#3]
Would be a nice way to ~dogfood~ validate the junit logger on itself! :)

**Comment by Siphonophora** [comment]
Ha, sounds good. I'm wiring up the pre-release package to some repositories at work and will see if I have any issues. I haven't used circle Ci, but can give that a shot when I have some time.

**Comment by Siphonophora** [comment]
@codito Since CircleCI is running in a linux container, It can't build the Net Framework project as part of building the acceptance testing project. This pipeline is a mess from trying to figure this out, but the issue is clear: https://circleci.com/gh/Siphonophora/junit.testlogger/12 I'd rather not change anything else about the acceptance testing proj just for this. 

I have tested the pre release package with a handful of different projects and haven't had any issues with stability. I found the bug resolved by #8 and a reproducible bug with GitLab's Junit parser which I reported to them. 

I'm comfortable with calling this stable and moving it to nuget. Any reason not to do that?

**Comment by codito** [comment]
@Siphonophora sounds great to me. One last todo :) Would you mind adding your nuget.org alias as owner to https://github.com/spekt/junit.testlogger/blob/master/src/package/JUnitXml.TestLogger.nuspec? I will push the next successful appveyor build to nuget.

I think circleci may work if we install mono similar to the travisci build. This doesn't block the nuget release.

Thanks for testing and the hard work!

**Comment by Siphonophora** [comment]
@codito All set. By the way the Travis CI build is acting like its broken

**Comment by codito** [comment]
:fireworks: https://www.nuget.org/packages/JUnitXml.TestLogger

Master build for travis ci seems to succeed. Please share the failed build link, I will take a look.
https://travis-ci.com/spekt/junit.testlogger

**Comment by Siphonophora** [comment]
This is what I meant. 
![image](https://user-images.githubusercontent.com/32316111/65652233-171d2d80-dfdf-11e9-817f-1c3e3b0ffe64.png)

**Comment by Siphonophora** [comment]
By the way, do I need to do something else to show up on the owner's list on the Nuget page?

In terms of next steps, I was going to see if I can get this added to docs for vstest, GitLab and CircleCI. Is there anything else that makes sense to try for?

**Comment by codito** [comment]
It's weird that nuget is not showing you as an owner. We used the same `publish to nuget via myget` mechanism used for other packages in spekt org. Have added your alias manually in nuget.org.

Fixed the appveyor build badge and nuget badge in readme. Good catch, missed correcting this when the forked repo was removed :-/

Absolutely makes sense to add in the docs for the CI systems. Here's the vstest docs: https://github.com/microsoft/vstest-docs/blob/master/docs/report.md. No more tasks on top of my mind :)
